### PR TITLE
Fix job for event-publisher-proxy

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Fix job for event-publisher-proxy

**Related issue(s)**

See https://github.com/kyma-project/kyma/issues/9381

### CODE REVIEW

**Review by marcobebway — DISMISSED** [review]

File: development/tools/go.mod [hunk 1]
```diff
@@ -34,7 +34,7 @@ require (
 	github.com/tidwall/gjson v1.6.0
 	github.com/vrischmann/envconfig v1.2.0
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
-	golang.org/x/tools v0.0.0-20200913032122-97363e29fc9b // indirect
+	golang.org/x/tools v0.0.0-20200917221617-d56e4e40bc9d // indirect
```
Comment (Ressetkk) [inline development/tools/go.mod:37]: Is changing the go.mod file really necessary there? Could you revert the change?

**Review by Ressetkk — CHANGES_REQUESTED** [review]
Is changing go.mod and sum really needed? If not then could you revert the change?

File: development/tools/go.mod (hunk 1 above)
Comment (k15r) [reply development/tools/go.mod]: this change constantly happens when you use the makefile to generate the jobs and execute the test. Imho this should be fixed on makefile level

Comment (sayanh) [reply development/tools/go.mod]: Done

**Review by marcobebway — APPROVED** [review]

**Review by Ressetkk — APPROVED** [review]
